uvm/servlets/admin/app/overrides/form/field/VTypes.js

```diff
@@ -243,11 +243,11 @@ Ext.define('Ung.overrides.form.field.VTypes', {
         case 'any':
             return true;
         default:
-            if (val.indexOf('>') !== -1 && val.indexOf(',') === -1) {
-                return this.isSinglePortValid( val.substring( val.indexOf('>') + 1 ));
+            if (val.indexOf('>=') !== -1 && val.indexOf(',') === -1) {
+                return this.isSinglePortValid( val.substring( val.indexOf('>=') + 2 ));
             }
-            if (val.indexOf('<') !== -1 && val.indexOf(',') === -1) {
-                return this.isSinglePortValid( val.substring( val.indexOf('<') + 1 ));
+            if (val.indexOf('<=') !== -1 && val.indexOf(',') === -1) {
```

Comment:
The val check for >=, >. <, and <= are all values the user is able to specify so we need to support those as well.  I think this code could break that, so be sure to retain support for the "non =" operations as well.

Reply:
Added code to handle `>` and `<` Symbols

![Screenshot from 2024-02-08 20-42-57](https://github.com/untangle/ngfw_src/assets/154422821/7cc9b618-e3e9-49d3-ae38-db6556a6a1ad)

![Screenshot from 2024-02-08 20-42-42](https://github.com/untangle/ngfw_src/assets/154422821/5a10d27e-b5d5-4b42-92ba-a7f0a5ece162)

![Screenshot from 2024-02-08 20-42-26](https://github.com/untangle/ngfw_src/assets/154422821/b22decf1-a68a-4ed7-80f6-d42aed5efd73)